app/views/angular/boot.js.erb

```diff
@@ -0,0 +1,21 @@
+var includeJs = function(url, callback) {
```

Comment:
All I have at this point is vanilla javascript and darn does it feel good. #stoneage